var/spack/repos/builtin/packages/py-esmpy/package.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyEsmpy(PythonPackage):
+    """ESMPy is a Python interface to the Earth System Modeling Framework (ESMF)"""
+
+    homepage = "https://earthsystemmodeling.org/"
+    git = "https://github.com/esmf-org/esmf.git"
```

Comment:
Can you add a url attribute too? It's used by `spack versions` and `spack checksum`.